### PR TITLE
feat: Docker compose

### LINKED ISSUE #918: Docker Compose support?
Hi,

I just found this awesome project. But I couldn't find way to make it up using Docker Compose. Does it possible to have _docker-compose.yml_ for this? That will be super helpful to spin up this explorer easier and faster in many environments.

Thank you. ❤️

### PR DESCRIPTION
**Pull request by dviramontes**
closes #918 

## Motivation

- Adds support for docker-compose

### Enhancements / other fixes
- fix `plug_cowboy` flags on dependency, required for prod build
- fixes node-sass dependency in linux with npm version bump

## Upgrading

- may require some users to run `npm install`

### CODE REVIEW

File: apps/indexer/config/dev/geth.exs [hunk 1]
```diff
@@ -6,7 +6,8 @@ config :indexer,
     transport: EthereumJSONRPC.HTTP,
     transport_options: [
       http: EthereumJSONRPC.HTTP.HTTPoison,
-      url: System.get_env("ETHEREUM_JSONRPC_HTTP_URL") || "https://mainnet.infura.io/8lTvJTKmHPCHazkneJsY",
+      url:
+        System.get_env("ETHEREUM_JSONRPC_HTTP_URL") || "https://mainnet.infura.io/v3/8a14134acf5d4d7ababe7a94484bbffb",
```
Comment (mvpratt) [inline apps/indexer/config/dev/geth.exs]: @dviramontes we shouldn't put our infura access code in the source code 😉

Comment (dviramontes) [reply apps/indexer/config/dev/geth.exs]: its not ours its my personal one

Comment (dviramontes) [reply apps/indexer/config/dev/geth.exs]: reverted

Comment (mvpratt) [reply apps/indexer/config/dev/geth.exs]: word

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      context: .
+    environment:
+      - PGHOST=postgres
+      - PGUSER=postgres
+      - PGPASSWORD=postgres
+      - ETHEREUM_JSONRPC_VARIANT=geth
+      - COIN=ETH
+      - MIX_ENV=prod
+      - PORT=4000
+      - SECRET_KEY_BASE=RMgI4C1HSkxsEjdhtGMfwAHfyT6CKWXOgzCboJflfSm4jeAlic52io05KB6mqzc5
```
Comment (0xMillz) [inline docker-compose.yml]: Committing secrets?

Comment (dviramontes) [reply docker-compose.yml]: all the time ;) `SECRET_KEY_BASE ` is for [cookie based auth](https://phoenixframework.org/blog/sessions) - the user is suppose to change it themselves before deploying. The one in there is just an example.

Comment (dviramontes) [reply docker-compose.yml]: Good catch though, i'll add a comment or something.

**Pull request merged**